### PR TITLE
No such file or directory creating .env file

### LINKED ISSUE #1414: [bug]: No such file or directory creating .env file
**Describe the bug**
Running the command yarn buildpack create-env-file packages/venia-concept to generate the .env file throws an error.

**To Reproduce**
1. Run the command yarn buildpack create-env-file packages/venia-concept

**Expected behavior**
```
$ yarn buildpack create-env-file packages/venia-concept
yarn run v1.16.0
$ /Users/jondoe/Sites/pwa-studio/node_modules/.bin/buildpack create-env-file packages/venia-concept
  ⓧ  Missing required environment variables:
     MAGENTO_BACKEND_URL: Connect to an instance of Magento 2.3 by specifying its public domain name. (eg.
     "https://master-7rqtwti-mfwmkrjfqvbjk.us-4.magentosite.cloud/")
  ⚠  The current environment is not yet valid; please set any missing variables to build the project before generating a .env file.
  ℹ  Successfully wrote a fresh configuration file to /Users/jondoe/Sites/pwa-studio/packages/venia-concept/.env
✨  Done in 0.41s.
```

**Actual**
```
yarn run v1.16.0
$ /Users/jondoe/Sites/pwa-studio/node_modules/.bin/buildpack create-env-file packages/venia-concept
fs.js:114
    throw err;
    ^

Error: ENOENT: no such file or directory, scandir '/Users/jondoe/Sites/pwa-studio/packages/pwa-buildpack/dist/cli'
    at Object.readdirSync (fs.js:790:3)
    at requireDirectory (/Users/jondoe/Sites/pwa-studio/node_modules/require-directory/index.js:59:6)
    at Object.addDirectory (/Users/jondoe/Sites/pwa-studio/packages/pwa-buildpack/node_modules/yargs/lib/command.js:112:33)
    at Object.Yargs.self.commandDir (/Users/jondoe/Sites/pwa-studio/packages/pwa-buildpack/node_modules/yargs/yargs.js:364:13)
    at Object.<anonymous> (/Users/jondoe/Sites/pwa-studio/packages/pwa-buildpack/bin/buildpack:4:6)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
**Screenshots**

**Additional context**

**Possible solutions**
1. Update the [ReadMe](https://github.com/magento-research/pwa-studio/blob/develop/pwa-devdocs/src/venia-pwa-concept/setup/index.md#step-2-install-pwa-studio-dependencies) to include yarn build after install or
2. Update this command to use src. I noticed this isn't included in #1404 so didn't know if it's as easy as `dist` to `src`.

**Please complete the following device information:**
 - Device [e.g. iPhone6, PC, Mac, Pixel3]: Mac
 - OS [e.g. iOS8.1, Windows 10]: macOS Mojave 10.14.5 (18F132)
 - Browser [e.g. Chrome, Safari]: -
 - Browser Version [e.g. 22]: -
 - Magento Version: dev-2.3-develop
 - PWA Studio Version:
 - NPM version `npm -v`: 2.15.11
 - Node Version `node -v`: v10.16.0

**Please let us know what packages this bug is in regards to:**
- [ ] `venia-concept`
- [x] `pwa-buildpack`
- [ ] `peregrine`
- [x] `pwa-devdocs`
- [ ] `upward-js`
- [ ] `upward-spec`

### PR DESCRIPTION
**Pull request by diazwatson**
## Description

#1414 Running the command `yarn buildpack create-env-file packages/venia-concept` to generate the .env file throws an error.

## Expected
```
$ yarn buildpack create-env-file packages/venia-concept
yarn run v1.16.0
$ /Users/jondoe/Sites/pwa-studio/node_modules/.bin/buildpack create-env-file packages/venia-concept
  ⓧ  Missing required environment variables:
     MAGENTO_BACKEND_URL: Connect to an instance of Magento 2.3 by specifying its public domain name. (eg.
     "https://master-7rqtwti-mfwmkrjfqvbjk.us-4.magentosite.cloud/")
  ⚠  The current environment is not yet valid; please set any missing variables to build the project before generating a .env file.
  ℹ  Successfully wrote a fresh configuration file to /Users/jondoe/Sites/pwa-studio/packages/venia-concept/.env
✨  Done in 0.41s.
```
## Actual
```
yarn run v1.16.0
$ /Users/jondoe/Sites/pwa-studio/node_modules/.bin/buildpack create-env-file packages/venia-concept
fs.js:114
    throw err;
    ^

Error: ENOENT: no such file or directory, scandir '/Users/jondoe/Sites/pwa-studio/packages/pwa-buildpack/dist/cli'
    at Object.readdirSync (fs.js:790:3)
    at requireDirectory (/Users/jondoe/Sites/pwa-studio/node_modules/require-directory/index.js:59:6)
    at Object.addDirectory (/Users/jondoe/Sites/pwa-studio/packages/pwa-buildpack/node_modules/yargs/lib/command.js:112:33)
    at Object.Yargs.self.commandDir (/Users/jondoe/Sites/pwa-studio/packages/pwa-buildpack/node_modules/yargs/yargs.js:364:13)
    at Object.<anonymous> (/Users/jondoe/Sites/pwa-studio/packages/pwa-buildpack/bin/buildpack:4:6)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

## Related Issue

Closes #1414 
## Verification Steps

1. Run the command `yarn buildpack create-env-file packages/venia-concept`
## Proposed Labels for Change Type/Package

- [ ] major (e.g x.0.0 - a breaking change)
- [ ] minor (e.g 0.x.0 - a backwards compatible addition)
- [x] patch (e.g 0.0.x - a bug fix)

## Checklist:

- [ ] I have updated the documentation accordingly, if necessary.
- [ ] I have added tests to cover my changes, if necessary.

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento-research/pwa-studio?pullRequest=1411) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento-research/pwa-studio?pullRequest=1411) before we can accept your contribution.<br/>

**Comment by sirugh** [comment]
Hey! Thanks for your contribution. Looks like you may have found a bug. Can you open an issue so we can track it properly? Make sure to include repro steps.

Additionally, at a glance I was wondering if you had run `yarn install` before attempting to run the buildpack command.

**Comment by diazwatson** [comment]
@sirugh I'll do

File: packages/pwa-buildpack/bin/buildpack [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env node
 
 require('yargs')
-    .commandDir('../dist/cli')
+    .commandDir('../src/cli')
```
[the remark this answers is not in the document]
Comment (sirugh) [reply packages/pwa-buildpack/bin/buildpack]: Also ping @jcalcaben, see above.

**Comment by diazwatson** [comment]
@sirugh Running `yarn build` after install seems to fix the issue.

**Pull request closed without merging**